### PR TITLE
added nudge for code_review_hotness metric threshold

### LINKED ISSUE #71: implement metrics-threshold-based nudges

### PR DESCRIPTION
**Pull request by andycmaj**
closes #71

### CODE REVIEW

File: consumers/src/handlers/processMetrics/codeReviewHotnessNudge.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  return {
+    userId,
+    teamId,
+    message: SlackRenderer.render(
+      <Message>
+        <SectionBlock>
+          <MarkdownText>
+            Hey <Mention userId={userDataSource.externalId} />
+            Hot code review alert! :thermometer:
```
Comment (davemeyer) [inline consumers/src/handlers/processMetrics/codeReviewHotnessNudge.tsx]: we probably need some process for organizing the emojis we use. and making sure that custom ones are added before trying to use them.

File: consumers/src/handlers/processMetrics/codeReviewHotnessNudge.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          <ButtonElement actionId="view" style="primary">
+            Get involved!
+          </ButtonElement>
+          <ButtonElement actionId="snooze">Snooze</ButtonElement>
+          <ButtonElement actionId="mute" style="danger">
+            Please stop!
+          </ButtonElement>
+        </ActionsBlock>
+        <ContextBlock>
+          <PlainText>This review has been open for N days.</PlainText>
```
Comment (davemeyer) [inline consumers/src/handlers/processMetrics/codeReviewHotnessNudge.tsx]: probably a TODO or new issue for replacing the `N` placeholder

File: consumers/src/handlers/processMetrics/codeReviewHotnessNudge.tsx (hunk 1 above)
Comment (andycmaj) [reply consumers/src/handlers/processMetrics/codeReviewHotnessNudge.tsx]: yeah is this a custom one?

Comment (andycmaj) [reply consumers/src/handlers/processMetrics/codeReviewHotnessNudge.tsx]: created https://github.com/asynchronous-dev/react-chat-renderer/issues/9

**Pull request merged**